docs/concepts/correlated-objects-pattern.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Avoid using the context.sync method in loops
+description: 'Learn how to use the split loop and correlated objects patterns to avoid calling context.sync in a loop.'
+ms.date: 04/09/2020
+localization_priority: Normal
+---
+
+
+# Avoid using the context.sync method in loops
+
+> [!NOTE]
+> This article assumes that you're beyond the beginning stage of working with at least one of the four host-specific Office JavaScript APIs &mdash; for Excel, Word, OneNote, and Visio &mdash; that use a batch system to interact with the Office document. In particular, you should know what a call of `context.sync` does and you should know what a collection object is. If you're not at that stage, please start with [Understanding the Office JavaScript API](../develop/understanding-the-javascript-api-for-office.md) and the documentation linked to under "host-specific" in that article.
```

Comment:
I thought there should not be spaces around an em dash? (https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28766)